src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -954,5 +971,16 @@ public NodeDialogEventArgs(NodeModel model)
             Handled = false;
         }
     }
+
+    /// <summary>
+    /// Extension method to check if a model exists in a group
+    /// </summary>
+    internal static class Extensions
+    {
+        public static bool CheckIfModelExistsInAGroup(this ObservableCollection<AnnotationModel> groups, Guid nodeGuid)
+        {
```

Comment:
It would probably be more flexible and accurate to use `IEnumerable<AnnotationModel>`.